moveit_ros/perception/handeye_calibration_solver/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+# Suppress the warning 'int _import_array()' defined but not used in 'usr/include/python2.7/numpy/__multiarray_api.h'
+add_compile_options(-Wno-unused-function)
+
+set(HEADERS
+  include/moveit/handeye_calibration_solver/handeye_solver_base.h
+  include/moveit/handeye_calibration_solver/handeye_solver_default.h
+)
+
+# ---[ Using cmake scripts and modules
+list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake/Modules)
+
+find_package(PythonLibs REQUIRED)
+find_package(NumPy REQUIRED)
```

Comment:
Is there a specific minimum version for either PythonLibs or NumPy?

Reply:
The minimum version number has been specified.